addons/supplies/functions/fnc_traitActions.sqf

```diff
@@ -0,0 +1,71 @@
+#include "script_component.hpp"
+/*
+ * Author: JoramD
+ * Add interactions for traits.
+ *
+ * Arguments:
+ * 0: Object <OBJECT>
+ * 1: Type <STRING>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [object] call tac_supplies_fnc_traitActions
+ *
+ * Public: No
+ */
+
+params ["_object", "_type"];
+
+private _removeTraitsAction = [
+    QGVAR(removeTraits),
+    LLSTRING(RemoveTraits),
+    "",
+    {
+        params ["", "_player"];
+
+        {
+            _player setUnitTrait [_x, false];
+        } forEach ["Medic", "Engineer", "ExplosiveSpecialist", "UavHacker"];
```

Comment:
You also want to send that to the server to remove them, otherwise reconnection will give them back.